content/blog/2022-07-07-out-in-the-open-june-2022.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Alles Fassade? – Out in the Open Juni 2022"
+author: "Klara Juhl - OKLab Osnabrück, Jörg Reichert - OKLab Leipzig, et al."
+date: '2022-07-07T12:00:00+02:00'
+excerpt: Monatlich erscheinender Überblick über Entwicklungen in der Open Data und Civic Tech Szene
+topic: community
+images:
+- imgname: out-in-the-open-2022-06-header.jpg
+  attribution: Foto von [Linnaea Mallette](https://publicdomainpictures.net/en/browse-author.php?a=18382) auf [publicdomainpictures.net](https://publicdomainpictures.net/en/view-image.php?image=265257&picture=neon-open-sign)
+og_image: out-in-the-open-2022-06-header.jpg
+type: blog
+---
+
+## Verschwörhaus
+
+Der im [letzten Out-In-The-Open-Blogbeitrag](https://codefor.de/blog/out-in-the-open-may-2022/) bereits erwähnte Konflikt um das Verschwörhaus Ulm ist weiter eskaliert: Nachdem die Ehrenamtlichen sich nicht darauf einlassen wollten, die jahrelang aufgebaute Marke "Verschwörhaus" (und den mit ihr verbundenen Werte-Versprechen) der Stadt zu überlassen, wurden sie nun aus den bisherigen Räumlichkeiten [geworfen](https://verschwoerhaus.de/das-verschwoerhaus-zieht-um/).
```

Comment:
https://gohugo.io/content-management/shortcodes/#ref-and-relref

Sorgt dafür, dass Hugo immer den richtigen Link hinschreibt

```suggestion
Der im [letzten Out-In-The-Open-Blogbeitrag]({{< ref "/content/blog/2022-06-09-out-in-the-open-may-2022.md" >}}) bereits erwähnte Konflikt um das Verschwörhaus Ulm ist weiter eskaliert: Nachdem die Ehrenamtlichen sich nicht darauf einlassen wollten, die jahrelang aufgebaute Marke "Verschwörhaus" (und den mit ihr verbundenen Werte-Versprechen) der Stadt zu überlassen, wurden sie nun aus den bisherigen Räumlichkeiten [geworfen](https://verschwoerhaus.de/das-verschwoerhaus-zieht-um/).
```